### PR TITLE
Updates to ecosystem.md do clarify community roles

### PR DESCRIPTION
**Pull request by lunaleaps**
Changelog: [Internal] - Update Ecosystem.md to clarify community roles

### CODE REVIEW

File: ECOSYSTEM.md
Comment (fortmarek) [inline ECOSYSTEM.md:29]: From our side, we could mention our opensource repositories like [flash-list](https://github.com/Shopify/flash-list) or [@shopify/react-native-skia](https://github.com/Shopify/react-native-skia) from William and Christian. We also sponsor SWM, among other things. @colinta, wdyt?

File: ECOSYSTEM.md
Comment (fortmarek) [inline ECOSYSTEM.md:19]: I don't know how we want to deal with cases when partners don't fulfil this requirement but we could decide on this when such a thing happens. I definitely think it's a great first step to make these requirements clear.

File: ECOSYSTEM.md
Comment (kelset) [inline ECOSYSTEM.md:52]: really like the idea of having a form for people to become core contributors 👍

File: ECOSYSTEM.md
Comment (kelset) [inline ECOSYSTEM.md:28]:
```suggestion
* **[Microsoft](https://twitter.com/ReactNativeMSFT):** Develops [React Native Windows](https://github.com/Microsoft/react-native-windows) and [React Native macOS](https://github.com/microsoft/react-native-macos) for building apps that target Windows and macOS; maintains [rnx-kit](https://github.com/microsoft/rnx-kit), [react-native-test-app](https://github.com/microsoft/react-native-test-app) and coordinates cross-companies efforts such as the [bundle working group](https://github.com/microsoft/rnx-kit/discussions/categories/bundle-working-group).
```

Adding a few more things to our list of efforts

File: ECOSYSTEM.md
Comment (kelset) [inline ECOSYSTEM.md:58]: not sure if here in the examples of where we aim for transparent and open communication you might want to add the releases WG (https://github.com/reactwg/react-native-releases) and the new arch WG (https://github.com/reactwg/react-native-new-architecture)

File: ECOSYSTEM.md
Comment (kelset) [inline ECOSYSTEM.md:17]: other potential options, just throwing there for brainstorming:
* help maintain the website and showcase (example of this right now: Simek from Expo)
* produce content/material/blogposts to showcase their usage of react-native on a consistent basis

Comment (lunaleaps) [reply ECOSYSTEM.md:17]: Great suggestions! One thing to clarify -- I was under the impression we really need more community commitments for release co-pilot. My concern is that if we make the baseline responsibilities too broad, then we might end up here again. The responsibilities of partners should be meeting baseline and in-addition demonstrating ownership. 

I think the points you listed fit well with demonstrating ownership-- we should also update Expo's list to include the website maintenance (cc @Simek if that's right by you)

File: ECOSYSTEM.md
Comment (lunaleaps) [reply ECOSYSTEM.md:19]: Yea, I'm not sure either but agree consensus on these responsibilities is a start and if a partner isn't able to meet this, we can have a discussion about why and I think that'll also be helpful

File: ECOSYSTEM.md
Comment (lunaleaps) [inline ECOSYSTEM.md:58]:
```suggestion
**We aim to make contributing to React Native as easy and transparent as possible.** We have discussion groups dedicated to the [new architecture rollout](https://github.com/reactwg/react-native-new-architecture), [releases](https://github.com/reactwg/react-native-releases), and [general questions and proposals](https://github.com/react-native-community/discussions-and-proposals). We are always looking for active, enthusiastic members of the React Native community to become core contributors.
```

File: ECOSYSTEM.md
Comment (brentvatne) [reply ECOSYSTEM.md:19]: the release co-pilot link anchor appears to be broken - i believe it should be updated to:

```suggestion
* Attending monthly meeting
* Contributing to the release process as a [community releaser](https://reactnative.dev/contributing/release-roles-responsibilities#release-role-2-community-releaser)
```

File: ECOSYSTEM.md
Comment (brentvatne) [inline ECOSYSTEM.md:25]:
```suggestion
* **[Expo](https://expo.dev/):** Builds [Expo Go and SDK](https://github.com/expo/expo), [Snack](https://snack.expo.dev/), and [Expo Application Services](https://expo.dev/eas). Maintains [React Native Directory](https://reactnative.directory/), stewards [React Navigation](https://reactnavigation.org/) along with other partners.
```

File: ECOSYSTEM.md
Comment (brentvatne) [reply ECOSYSTEM.md:19]: expo can certainly help out as a community releaser/release co-pilot in the future, although i wonder if it is useful to require that every partner contribute in this particular way. for example, our role in recent releases has been to provide our feedback from integrating the new release into our tooling and also give technical release support in the form of pull requests and reviews for pull requests. @kudo has been handling this.

File: ECOSYSTEM.md
Comment (brentvatne) [reply ECOSYSTEM.md:17]: @Simek has been contributing to the website off-the-clock, but we'll discuss whether we want to possibly make this an official responsibility. i won't have an answer soon and we shouldn't hold up this doc on it

Comment (kelset) [reply ECOSYSTEM.md:17]: yeah my point is basically related to what @brentvatne is saying in his other comment about ways of being involved:

> expo can certainly help out as a community releaser/release co-pilot in the future, although i wonder if it is useful to require that every partner contribute in this particular way. for example, our role in recent releases has been to provide our feedback from integrating the new release into our tooling and also give technical release support in the form of pull requests and reviews for pull requests. @Kudo has been handling this.

But yes, we'd like to get a couple more people that can be part of the release crew from the community perspective. I'm just concerned that some companies might shy away from becoming a partner if we set that as an "hard requirement"

Comment (lunaleaps) [reply ECOSYSTEM.md:17]: I see, I defer then to you guys what you think is necessary to state as baseline requirements -- or if there should be any at all. My impression is that it is difficult to get consistent commitment from the community for release support and leveraging our partners (which of course they're already doing) could help. 

We could make the baseline requirement still "support releases" but in a more broad definition as @brentvatne shared or get rid of it.

Comment (brentvatne) [reply ECOSYSTEM.md:17]: for this pass it may be appropriate to keep the definition broad, and we can refine that as we gain a better understanding of where different partners can contribute, and what we think should be a fair contribution level to balance the upsides of being a partner organization

**Comment by kelset** [comment]
sidenote: I wonder if we should talk with Wix about if they want to be a partner: after all, they have been using RN for a really long time, and maintain quite a few key libraries in the ecosystem like [detox](https://github.com/wix/detox) and [react native navigation](https://github.com/wix/react-native-navigation)

File: ECOSYSTEM.md
Comment (lunaleaps) [reply ECOSYSTEM.md:52]: @cortinico's idea!

**Comment by asafkorem** [comment]
Hey, Asaf from Wix here (a maintainer of Detox),

I totally agree with @kelset, Wix is a great fit under the definition of "RN partners", we maintain "detox", "react-native-navigation", "react-native-calendars", "react-native-ui-lib" and many more popular RN open sources. 
You can check https://github.com/wix?q=react-native for more examples.

I'd also say that we are very connected to the RN community, our guys were always speaking at conferences and podcasts related to RN and mobile development, writing blog posts, etc.

File: ECOSYSTEM.md
Comment (lunaleaps) [reply ECOSYSTEM.md:29]:
```suggestion
* **[Shopify](TODO):** Maintains [flash-list](https://github.com/Shopify/flash-list) and [@shopify/react-native-skia](https://github.com/Shopify/react-native-skia) and sponsorship of Software Mansion. 
```

Comment (lunaleaps) [reply ECOSYSTEM.md:29]: @fortmarek @colinta Does that suggestion look right? Is there a preferred link?

File: ECOSYSTEM.md
Comment (scarlac) [inline ECOSYSTEM.md:31]:
```suggestion
* **[Tesla](https://www.tesla.com):** Maintains [React Native Camera Kit](http://github.com/teslamotors/react-native-camera-kit) and contributes to React Native core as well as various community libraries.
```

File: ECOSYSTEM.md
Comment (jamonholmgren) [inline ECOSYSTEM.md:23]: Is Coinbase going to continue to invest in this? I know at least some of their internal RN champions were laid off earlier this year.

File: ECOSYSTEM.md
Comment (jamonholmgren) [inline ECOSYSTEM.md:26]: IMO it's worth noting RN Radio podcast (5k+ subscribers) and RN Newsletter (13k+ subscribers), as the largest of their respective kinds in the RN community.

```suggestion
* **[Infinite Red](https://infinite.red/):** Maintains the [ignite cli/boilerplate](https://github.com/infinitered/ignite), organizes [Chain React Conf](https://cr.infinite.red/), hosts the [React Native Radio podcast](https://reactnativeradio.com), publishes the [React Native Newsletter](https://reactnativenewsletter.com)
```

File: ECOSYSTEM.md
Comment (lunaleaps) [reply ECOSYSTEM.md:23]: Yea that's my question as well. We can discuss this in our next meeting. I think if I don't hear word back, will remove them from this list

**Comment by brentvatne** [comment]
pending the TODO that we'll resolve in the next sync, this looks shippable to me!

**Review by cipolleschi — APPROVED** [review]

File: ECOSYSTEM.md
Comment (thymikee) [inline ECOSYSTEM.md:24]:
```suggestion
* **[Callstack](https://callstack.com/):** Maintains the [React Native CLI](https://github.com/react-native-community/react-native-cli) and [other community libraries](https://github.com/callstack), organizes [React Native EU](https://react-native.eu/) and hosts [The React Native Show podcast](https://www.callstack.com/podcast-react-native-show)
```

**Review by thymikee — COMMENTED** [review]
Agree with other comments and the doc looks good to me in this shape :shipit:

File: ECOSYSTEM.md
Comment (brentwalter) [reply ECOSYSTEM.md:23]: 👋 Hi team! Brent here from Coinbase. We're down, but not out 😄 .

We're ramping back up, and hopefully y'all have been seeing Bruno in the weekly RN release meetings.

Comment (Zizzamia) [reply ECOSYSTEM.md:23]: Also, it's worth mentioning we are getting close to open-source some of our tooling around measuring React Native performance.

For now, we shared a bit of a tease in this recent article https://blog.coinbase.com/performance-vitals-a-unified-scoring-system-to-guide-performance-health-and-prioritization-127f80268ee4, and more to come soon 😁

**Review by asafkorem — COMMENTED** [review]
Hi there! Asaf from Wix here 🙂 
Having joined Wix as a partner, we look forward to being listed here alongside our most popular (and maintained) React Native open sources:
- [Detox](https://wix.github.io/Detox/)
- [React Native Navigation](https://wix.github.io/react-native-navigation/)
- [React Native UILib](https://wix.github.io/react-native-ui-lib/)
- [React Native Calendars](https://wix.github.io/react-native-calendars/)
- [React Native Notifications](https://github.com/wix/react-native-notifications)

Basically we have 32 public repos for react native, most of them are open source projects, see on GitHub: https://github.com/orgs/wix/repositories?q=react-native

File: ECOSYSTEM.md
Comment (asafkorem) [inline ECOSYSTEM.md:32]:
```suggestion
* **[Wix.com](wix.engineering/open-source):** Maintains a variety of React Native open source projects ([see all](https://github.com/orgs/wix/repositories?q=react-native)), including:  [Detox](https://wix.github.io/Detox/) end-to-end testing library for React Native apps, [RN UILib](https://wix.github.io/react-native-ui-lib/), [RN Navigation](https://wix.github.io/react-native-navigation/), [RN Calendars](https://wix.github.io/react-native-calendars/) and [RN Notifications](https://github.com/wix/react-native-notifications).

```

File: ECOSYSTEM.md
Comment (capezzbr) [reply ECOSYSTEM.md:23]: Additional work we are driving:
- Helping out migrating `@react-native-community/datetimepicker` to be Fabric compatible (and possibly others modules in the upcoming months)
- Investigating enabling the new architecture in our RN app, and will be happy to write a blogpost documenting the process and results once the project is completed
- Can help out with

Comment (capezzbr) [reply ECOSYSTEM.md:23]: Hey @lunaleaps, here our suggestion:

> **[Coinbase](https://coinbase.com/):** Builds world class crypto apps in React Native. Publishes [articles](https://blog.coinbase.com/tagged/react-native) to advertise React Native development. Supports OSS libraries on their migration to the new architecture.

**Comment by kelset** [comment]
Now that this conversation has been going on for a while, I wanted to check back on the partners' requirements aspect of this document; see [this comment thread](https://github.com/facebook/react-native/pull/34341#discussion_r937448097) and [this comment thread](https://github.com/facebook/react-native/pull/34341#discussion_r937665680) for prior conversations. (cc @brentvatne and @lunaleaps since we were having those conversations with them)

Here are some personal perspectives on this that I'd like to share and we can discuss if/what to integrate:
* only a company can be a partner
  * and defines 2 "main points of contact" people
* there should be some "hard" requirements; if a company doesn't live up to them, they are released from the partner role (become a "partner alumni" or something like that, if they have been for a while and then not)
* I think the requirements should be "two-folds", one towards contributing ("inward") and one towards advocacy/community ("outward"). Partners should be both actively contributing to the ecosystem's progress and advocate/talk about it with the broader community.
  * in my head, this would look something like having two lists and to be a partner you need to satisfy one or more items per each list. Here are some examples of what I'm thinking about, feel free to suggest more:
  * contributing ("inward")
    * part of the release crew (see [community releaser](https://reactnative.dev/contributing/release-roles-responsibilities#release-role-2-community-releaser) for details)
    * actively maintains at least one library in the ecosystem
    * frequently contributes to core libraries (frequency TBD - at least a few times each month?)
    * maintains react-native target platforms (ex. win, macos, tvos, etc)
    * coordinates/drives working groups or efforts to improve the ecosystem (ex. see https://github.com/react-native-community/developer-experience-wg)
  * advocacy ("outward")
    * writes blogposts around react native, using react native (frequency TBD - at least once per quarter?)
    * has an active social media presence (ex. twitter account from their engineering) that talks about RN
    * contributes frequently to the reactnative.dev website and documentation
    * organises conferences/meetups
    * produces a podcast, newsletter, etc similar content
    * speaks at confs, podcasts (frequency TBD - at least once per quarter?)
* being a partner unlocks a series of benefits:
  * attendance to the monthly meeting with the other partners + meta
    * expectation is that at least of of the two 2 "main points of contact" people will always attend
  * access to dedicated channels in the Discord server
  * prominence in the showcase (details TBC - I think we need a "lower limits" of apps from the partner before they have their own "row" https://reactnative.dev/showcase)

That's all that I can think of for now - again, this is just a personal perspective on this. Would love to discuss it further so that we can make this document progress forward. I think that the clarity around requirements will also make the process of becoming a partner more clear and help other companies also consider it as an option.

**Comment by lunaleaps** [comment]
@kelset thanks for sharing your thoughts. Sorry for the lack of clarity here but my next step here was to figure out how a way to do intake for the Ecosystem form. I think the draft here is good as is and plan to put this into PR state once I get the workflow set up. I will be removing Tesla as they can't commit right now to the responsibilities we've highlighted.

I like the concrete list of ways to contribute and I think that's something we can add to this.

File: ECOSYSTEM.md
Comment (asafkorem) [inline ECOSYSTEM.md:31]:
```suggestion
* **[Wix.com](https://wix.engineering/open-source):** Maintains a variety of React Native open source projects ([see all](https://github.com/orgs/wix/repositories?q=react-native)), including:  [Detox](https://wix.github.io/Detox/) end-to-end testing library for React Native apps, [RN UILib](https://wix.github.io/react-native-ui-lib/), [RN Navigation](https://wix.github.io/react-native-navigation/), [RN Calendars](https://wix.github.io/react-native-calendars/) and [RN Notifications](https://github.com/wix/react-native-notifications).
```

**Pull request closed without merging**
